CHANGELOG.md

```diff
@@ -1,58 +1,95 @@
 # Changelog
+
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+## [0.3.1]
```

Comment:
just out of curiosity - did we stop adding dates on purpose or did we just forget about it?

Reply:
dates started to get inconsistent between github and changelog as tagging never happens on the day of the PR... so i actually stopped adding them.. not sure how to deal with this... i personally would just vote for not adding them to the changelog..